entrypoint.sh

```diff
@@ -34,29 +36,39 @@ set -e
 : ${GANESHA_OPTIONS:="-N NIV_EVENT -L STDOUT"}
 : ${GANESHA_EPOCH:=''} # For restarting
 
+if [ ! -z "${KV_CA_CERT}" ]; then
```

Comment:
I agree. But bringing this in doesn't change that. I was only bringing it in since I was merging other stuff anyway and didn't want to fall too far behind ceph-docker. I can remove it if you prefer.